CONTRIBUTING.md

```diff
@@ -9,6 +9,7 @@ We adhere to the [Contributor Covenant Code of Conduct](#contributor-covenant-co
 - [create a Draft Pull Request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request) with `WIP:` in its name to the `master` branch (example: `WIP: documentation updates`)
 - commit all your work to your new branch; other developers can provided feedback by reviewing your pull request
 - when the feature is ready: remove the `WIP:` and set it ready to review
+- when your changes get approved, remember to always select **Squash and merge** from the drop-down button
```

Comment:
Good, but after talking to you, i changed github settings and this is now enforced by github.

Reply:
FYI: since you are doing a lot of doc changes in this PR, it might be handy to know that you can skip CI builds by adding special keyword in your commit message. e.g.
```cmd
git commit -m 'documentation update [ci skip]'
```
will make travis skip the build.

Reply:
I will let the **Squash and merge** option for reference and also add the ci skip